sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -426,14 +426,15 @@ multilib_src_install_all() {
 	# Flatcar: Don't enable services in /etc, move to /usr.
 	systemd_enable_service multi-user.target systemd-networkd.service
 	systemd_enable_service multi-user.target systemd-resolved.service
+	systemd_enable_service multi-user.target remote-fs.target
 	systemd_enable_service sysinit.target systemd-timesyncd.service
 
 	# Flatcar: Enable getty manually.
 	mkdir --parents "${ED}/usr/lib/systemd/system/getty.target.wants"
 	dosym ../getty@.service "/usr/lib/systemd/system/getty.target.wants/getty@tty1.service"
 
-	# Flatcar: Do not enable random services if /etc was detected
-	# as empty!!!
+	# Flatcar: Use an empty preset file, because systemctl
+	# preset-all puts symlinks in /etc, not in /usr.
```

Comment:
Good idea to make the reason more clear. Maybe we can also add that we don't use `/etc` because it's not auto-updated.

Reply:
Good idea, will update it.

Reply:
Comment updated.